tests/dashbio_demos/app_ideogram.py

```diff
@@ -14,833 +15,750 @@
-                                                        "value": "vertical",
-                                                    },
-                                                    {
-                                                        "label": "Horizontal",
-                                                        "value": "horizontal",
-                                                    },
-                                                ],
-                                                value="horizontal",
-                                            ),
-                                            html.P("Hover Data (Overlay-1/2)"),
-                                            html.Div(
-                                                children=[
-                                                    html.Span(
-                                                        "None",
-                                                        id="annote-data",
-                                                        style={
-                                                            "color": "#0D76BF"},
-                                                    )
-                                                ],
-                                                className="ideogram-databox-parameters",
-                                            ),
-                                        ]
-                                    ),
-                                ],
-                                className="two columns ideogram-column",
-                            ),
-                            html.Div(
-                                [
-                                    dash_bio.Ideogram(
-                                        id="ideo-annotations",
-                                        dataDir="https://unpkg.com/ideogram@1.3.0/"
-                                                "dist/data/bands/native/",
-                                        organism="human",
-                                        assembly="GRCh37",
-                                        orientation="horizontal",
-                                        showBandLabels=True,
-                                        chrHeight=275,
-                                        chrMargin=28,
-                                        rotatable=True,
-                                        filterable=True,
-                                        className="ideogram-custom",
-                                    )
-                                ],
-                                className="ten columns",
-                            ),
-                        ],
-                    ),
+options = {
+    'custom': [
+        html.H4('Organism'),
+        html.Div(className='ideogram-controls-name', children='Species'),
+        dcc.Dropdown(
+            className='ideogram-dropdown',
+            id='organism-change',
+            options=[
+                {
```

Comment:
Maybe all the below could be written as an ndjson comprehension... Just kidding, it's probably not worth it, but I came across this lib: https://jsonlines.readthedocs.io/